generators/generator-base-private.js

```diff
@@ -865,8 +866,7 @@ module.exports = class extends Generator {
             this.warning(msg);
             return undefined;
         }
-        // eslint-disable-next-line global-require,import/no-dynamic-require
-        return require(path.join(blueprintPackagePath, 'package.json'));
+        return fs.readJsonSync(path.join(blueprintPackagePath, 'package.json'));
```

Comment:
it's not needed to have fs-extra as dep, this line can be replaced by
  - readFileSync
  - JSON.parse

Reply:
fs-extra is already a prod deep for jhipster-core. https://github.com/jhipster/generator-jhipster/blob/377ee744714055e8eaf364def484b84766246a01/package-lock.json#L3807

Have you removed it?

Reply:
this is taken care of in the merged version, I've decided to keep it for now, but I'm changing the code to remove it because it's not necessary